cds/modules/records/providers.py

```diff
@@ -48,37 +49,6 @@ class CDSRecordIdProvider(BaseProvider):
     default_status = PIDStatus.RESERVED
     """Record IDs are by default registered immediately."""
 
-    @classmethod
-    def create(cls, object_type=None, object_uuid=None, **kwargs):
-        """Create a new record identifier."""
-        # Request next integer in recid sequence.
-        assert 'pid_value' not in kwargs
-
-        provider_url = current_app.config.get('RECORDS_ID_PROVIDER_ENDPOINT',
```

Comment:
I would suggest deleting this var from the config and add a card to the board to remove it from deployment too.

Reply:
done!